tests/dashbio_demos/dash-alignment-viewer/assets/base-app-page.css

```diff
@@ -0,0 +1,248 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400');
```

Comment:
Is there a way to keep this at a higher level within the directory and then copy it into the folder containing the app that's being deployed? I think that might be better than having to copy/paste this file into every single app's folder.

Reply:
I don't know if the standalone `index.py` at the repo root will work properly if we do that. I'll take a crack at it though, since I agree, the copy-pasta doesn't feel great.

Reply:
We could probably get rid of that. It's really only around for heroku test deployments and I personally haven't looked at it in a long time now that we have integration/visual tests. CC @mkcor since she would have some more valuable input into this issue!

Reply:
One `assets/` folder per demo app looks tedious indeed... but I can see how it might be necessary to make each app actually standalone... I'm not sure, I would need to dig deeper.